doc/ChangeLog

```diff
@@ -1,4 +1,113 @@
 ===============================================================
+Tag name:  ctsm1.0.dev029
+Originator(s):  slevis (Samuel Levis, SLevis Consulting LLC, 303-665-1310)
+Date:  Tue Feb 26 22:17:28 MST 2019
+One-line Summary: Collapse landunits to the N most dominant
+
+Purpose of changes
+------------------
+
+ Allows the collapse to fewer landunits using the concept of 
+ N most dominant for the purpose of faster runs, eg in the context
+ of NWP (Numerical Weather Prediction).
+
+Bugs fixed or introduced
+------------------------
+
+Issues fixed (include CTSM Issue #): #581
+
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm5_0
+
+[ ] clm4_5
+
+Notes of particular relevance for users
+---------------------------------------
+
+Caveats for users (e.g., need to interpolate initial conditions):
+
+Changes to CTSM's user interface (e.g., new/renamed XML or namelist variables):
+ New namelist variable n_dom_landunits with valid integer values from 0 to
+ max_lunit currently equal to 9, where 0 means “do nothing” and other
+ values determine the number of active landunits when running the model.
+
+Changes made to namelist defaults (e.g., changed parameter values):
```

Comment:
@slevisconsulting for template header line ChangeLog, go ahead and put "none" or "not applicable" or something if it doesn't apply. Same for tests that you didn't do. Although in that case I often delete those lines.